distribution/debian/scripts/preinst

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Add vector:vector user & group
-useradd -M --system --user-group vector
+useradd -M --system --user-group vector || true
```

Comment:
I'm a bit concerned that this command could fail for other reasons that we might actually want to abort from. Maybe we could wrap it in a conditional check for the `vector` user's existence?

Just for my edification is there a reason we switched from `adduser`?

Reply:
Sure.

`adduser` isn't present on all the platforms where people use the *.deb packages - useradd is lower-level and more widely supported.